tests/test_main.py

```diff
@@ -461,30 +451,19 @@ def test_main_dotnet(_1c444_dotnetfile_extractor):
 
 
 def test_main_dotnet2(_692f_dotnetfile_extractor):
-    # tests rules can be loaded successfully and all output modes
+    # tests successful execution and one rendering
+    # above covers all output modes
     path = _692f_dotnetfile_extractor.path
     assert capa.main.main([path, "-vv"]) == 0
-    assert capa.main.main([path, "-v"]) == 0
-    assert capa.main.main([path, "-j"]) == 0
-    assert capa.main.main([path, "-q"]) == 0
-    assert capa.main.main([path]) == 0
 
 
 def test_main_dotnet3(_0953c_dotnetfile_extractor):
-    # tests rules can be loaded successfully and all output modes
+    # tests successful execution and one rendering
     path = _0953c_dotnetfile_extractor.path
-    assert capa.main.main([path, "-vv"]) == 0
-    assert capa.main.main([path, "-v"]) == 0
     assert capa.main.main([path, "-j"]) == 0
-    assert capa.main.main([path, "-q"]) == 0
-    assert capa.main.main([path]) == 0
 
 
 def test_main_dotnet4(_039a6_dotnetfile_extractor):
-    # tests rules can be loaded successfully and all output modes
+    # tests successful execution and one rendering
     path = _039a6_dotnetfile_extractor.path
-    assert capa.main.main([path, "-vv"]) == 0
-    assert capa.main.main([path, "-v"]) == 0
-    assert capa.main.main([path, "-j"]) == 0
-    assert capa.main.main([path, "-q"]) == 0
     assert capa.main.main([path]) == 0
```

Comment:
im a little concerned that its not obvious that we're testing both the same and rendering mode. i know you have the comment, but im afraid i wouldn't remember what it means in a year or two. can we include more commentary?

Reply:
or maybe just use the same output mode for all of these, since we have the comprehensive output mode test above?